### DISCUSSION TITLE
[New Feature] Toggle: Aircraft always flies in from map edge

### DISCUSSION THREAD

**Discussion by askhati** [Vinifera-Developers/Vinifera#1446] [Feature Requests]
### Description:
In TS, if you build an aircraft while there are no empty helipads available, the newly-made aircraft will fly in from the map edge instead of appearing on the helipad. It would be useful to have an option where a certain aircraft type is ALWAYS brought in from the map edge, even if there are empty helipads available.

### Possible Implementation:
New tag for AircraftType: "FliesIn=yes/no" (defaults to no). If set to yes on the AircraftType, this aircraft will always spawn on the map edge and then fly in to the nearest player-controlled aircraft factory, even if there are empty helipads available.

Mod uses: for "building" large aircraft that would look comically large if they are built and then "appear" on a small helipad; instead, have them fly in from the map edge as "battlefield reinforcements". Possible examples: building a player-controlled version of the GDI Kodiak.

**Comment by SlavackGHUB** [comment]
Would this also prevent the aircraft cloning bug?

**Comment by JoyfulShush** [comment]
On the other hand, a different case that may happen is that your aircraft will be shot down as it comes in if the map edge is controlled by an enemy house. For example, in maps where the player is in the center of the map, surrounded by enemies.

It would be nice if that key, or a different one, would also prevent that behavior even if all Helipads are full, instead spawning the aircraft in a nearby free cell.
